src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -84,57 +86,74 @@
 
-    @Option(shortName = "CAP", doc = "Treat positions with coverage exceeding this value as if they had coverage at this value (but calculate the difference for PCT_EXC_CAPPED).", overridable = true)
+    @Argument(shortName = "CAP", doc = "Treat positions with coverage exceeding this value as if they had coverage at this value (but calculate the difference for PCT_EXC_CAPPED).")
     public int COVERAGE_CAP = 250;
 
-    @Option(doc="At positions with coverage exceeding this value, completely ignore reads that accumulate beyond this value (so that they will not be considered for PCT_EXC_CAPPED).  Used to keep memory consumption in check, but could create bias if set too low", overridable = true)
+    @Argument(doc="At positions with coverage exceeding this value, completely ignore reads that accumulate beyond this value (so that they will not be considered for PCT_EXC_CAPPED).  Used to keep memory consumption in check, but could create bias if set too low")
     public int LOCUS_ACCUMULATION_CAP = 100000;
 
-    @Option(doc = "For debugging purposes, stop after processing this many genomic bases.")
+    @Argument(doc = "For debugging purposes, stop after processing this many genomic bases.")
     public long STOP_AFTER = -1;
 
-    @Option(doc = "Determines whether to include the base quality histogram in the metrics file.")
+    @Argument(doc = "Determines whether to include the base quality histogram in the metrics file.")
     public boolean INCLUDE_BQ_HISTOGRAM = false;
 
-    @Option(doc="If true, count unpaired reads, and paired reads with one end unmapped")
+    @Argument(doc="If true, count unpaired reads, and paired reads with one end unmapped")
     public boolean COUNT_UNPAIRED = false;
 
-    @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
+    @Argument(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
-    @Option(doc = "If true, fast algorithm is used.")
+    @ArgumentCollection
+    protected IntervalArgumentCollection intervalArugmentCollection = getIntervalArgumentCollection();
+
+    @Argument(doc = "If true, fast algorithm is used.")
     public boolean USE_FAST_ALGORITHM = false;
 
-    @Option(doc = "Average read length in the file. Default is 150.", optional = true)
+    @Argument(doc = "Average read length in the file. Default is 150.", optional = true)
     public int READ_LENGTH = 150;
 
-    @Option(doc = "An interval list file that contains the positions to restrict the assessment. Please note that " +
-            "all bases of reads that overlap these intervals will be considered, even if some of those bases extend beyond the boundaries of " +
-            "the interval. The ideal use case for this argument is to use it to restrict the calculation to a subset of (whole) contigs.",
-            optional = true, overridable = true)
     public File INTERVALS = null;
```

Comment:
Same here .. I think this needs to be protected.  If anyone uses this class programmatically from another tool, and sets `INTERVALS` directly, it'll get clobbered by the value from `intervalArugmentCollection` via the assignment on line 450.  Better to get a compile break than a runtime break I think.